bokehjs/test/unit/core/util/svg.ts

```diff
@@ -0,0 +1,108 @@
+import {expect} from "assertions"
+
+import {SVGRenderingContext2D} from "@bokehjs/core/util/svg"
+
+function stringToHTML(str: string) {
+  const parser = new DOMParser()
+  const doc = parser.parseFromString(str, "text/html")
+  return doc.body
+}
+
+function compare_on_dom(fn: (ctx: CanvasRenderingContext2D) => void, svg: string, {width, height}: {width: number, height: number}) {
+  const canvas = document.createElement("canvas")
+  canvas.height = height
+  canvas.width = width
+
+  document.body.appendChild(canvas)
+
+  const ctx = canvas.getContext("2d") as CanvasRenderingContext2D
```

Comment:
If this cast is all about dropping `null`, then use `!` postfix operator instead.